### PR TITLE
Use explicit location in tests which use AddCredentialStore

### PR DESCRIPTION
**Pull request by olukas**
Since WFLY 13 location for AddCredentialStore is required when JCEKS type is used. In previous version WildFly is able to use some magic and generate location with name derived from resource name. However for Creaper command test purposes it is safer to use some named location.

### CODE REVIEW

File: testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/credentialstore/AddCredentialStoreOnlineTest.java [hunk 1]
```diff
@@ -24,13 +29,26 @@
     private static final String TEST_CREDENTIAL_STORE_NAME2 = "CreaperTestCredentialStore2";
     private static final Address TEST_CREDENTIAL_STORE_ADDRESS2 = SUBSYSTEM_ADDRESS
             .and("credential-store", TEST_CREDENTIAL_STORE_NAME2);
+    private static final String PATH_ADDRESS_NAME = "tmp.path";
+    private static final Address TMP_PATH_ADDRESS = Address.of("path", PATH_ADDRESS_NAME);
 
     private static final String TEST_PASSWORD = "somePassword";
 
+    private static String tmpFilePath;
```
Comment (mchoma) [inline testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/credentialstore/AddCredentialStoreOnlineTest.java]: why is this necessary? and TemporaryFolder tmp is not enough?

Comment (mchoma) [reply testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/credentialstore/AddCredentialStoreOnlineTest.java]: Is this just shortcut for tmp.getRoot().getAbsolutePath() ?

Comment (olukas) [reply testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/credentialstore/AddCredentialStoreOnlineTest.java]: yes, I want to avoid multiple `tmp.getRoot().getAbsolutePath()`
